src/sql/workbench/api/browser/mainThreadNotebookDocumentsAndEditors.ts

```diff
@@ -329,7 +328,8 @@ export class MainThreadNotebookDocumentsAndEditors extends Disposable implements
 		@IEditorService private _editorService: IEditorService,
 		@IEditorGroupsService private _editorGroupService: IEditorGroupsService,
 		@ICapabilitiesService private _capabilitiesService: ICapabilitiesService,
-		@INotebookService private readonly _notebookService: INotebookService
+		@INotebookService private readonly _notebookService: INotebookService,
+		@IFileService private readonly fileService: IFileService
```

Comment:
please use an underscore before the variable name to stay consistent with other private variables here